manoseimas/mps_v2/views/json.py

```diff
@@ -140,6 +146,26 @@ def suggesters_json(request):
                          'subtab_counts': subtab_counts()})
 
 
+def _mangle(text):
+    text = text.lstrip()
+    text = re.sub(u'^(PAGRINDINIO )?(KOMITETO )?(IŠVADA )?', u'', text)
+    text = text.lstrip()
```

Comment:
You could merge the two `lstrips()` with the regexp:

`re.sub(ur'^\s*(PAGRINDINIO )?(KOMITETO )?(IŠVADA )?\s*', u'', text)`.